velox/common/caching/SsdFile.h

```diff
@@ -366,10 +370,6 @@ class SsdFile {
   // Verifies that 'entry' has the data at 'run'.
   void verifyWrite(AsyncDataCacheEntry& entry, SsdRun run);
 
-  // Deletes checkpoint files. If 'keepLog' is true, truncates and syncs the
-  // eviction log and leaves this open.
-  void deleteCheckpoint(bool keepLog = false);
```

Comment:
If this is not supposed to be used by production code, then let's have a testing wrapper? testingDeleteCheckpoint? Thanks!

Reply:
it's used by prod code